bindings/wasm/examples/worker.ts

```diff
@@ -95,15 +95,13 @@ function addMembers(
   const cls = module[className];
   const obj = areStatic ? cls : cls.prototype;
   for (const name of methodNames) {
-    if (name != 'cylinder') {
```

Comment:
@geoffder Can I assume this was a debugging leftover of some kind?

Reply:
Yea looks like it sorry!